src/openlcb/Bootloader.hxx

```diff
@@ -1044,6 +1046,17 @@ bool bootloader_init() {
 /// should keep running (i.e., to call again).
 bool bootloader_loop()
 {
+    if (state_.request_reinit_node)
+    {
+        if (state_.init_state == INITIALIZED)
+        {
+            // Back out of the initialized state so that it can try the node
+            // initialized transition again. Assume the alias is still good. If
+            // it is not, then there is already recovery logic for this.
+            state_.init_state = static_cast<InitState>(state_.init_state - 1);
```

Comment:
THis would be better written with the actual enum of the state you want to go into. The current code is "just back one" which likely does not mean "send node initialized message" anymore if the code changes.